### PR TITLE
Jason cheng (#1 #4)

### PR DESCRIPTION
**Pull request by JasonCheng1**
Fixed Issue: Hello World? and Noisy Console!

### CODE REVIEW

File: challenge/src/App.jsx [hunk 1]
```diff
@@ -7,7 +7,7 @@ class App extends Component {
     state = {}
     constructor(props) {
         super(props);
-        console.log("Hello from the constructor!");
+        console.log();
```
Comment (mjrb) [inline challenge/src/App.jsx]: this almost works, i think this still prints a newline.

**Review by mjrb — CHANGES_REQUESTED** [review]
just a small tweak

File: challenge/src/App.jsx (hunk 1 above)
Comment (shivanmodha) [reply challenge/src/App.jsx]: yup

**Comment by mjrb** [comment]
@jasonCheng1 when you get a chance, could you just remove the console.log entirely, and then I can merge this in.
Thanks

**Comment by JasonCheng1** [comment]
@mjrb Still learning git, I forgot to push my changes a while back

**Comment by mjrb** [comment]
Np, learning is always good

**Review by mjrb — APPROVED** [review]
badda bing

**Pull request merged**